ttnn/cpp/ttnn/operations/data_movement/pad/device/pad_program_factory.cpp

```diff
@@ -182,7 +182,7 @@ operation::ProgramWithCallbacks pad_rm_opt(const Tensor &a,
     TT_ASSERT(unpadded_row_size_nbytes <= padded_row_size_nbytes, "Padded output tensor size should be >= input tensor size");
 
     Device *device = a.device();
-    auto dst_buffer_l1 = Buffer(device, padded_row_size_nbytes, padded_row_size_nbytes, BufferType::L1);
+    auto dst_buffer_l1 = Buffer::create(device, padded_row_size_nbytes, padded_row_size_nbytes, BufferType::L1);
```

Comment:
was it some bug?

Reply:
No, this PR removes the Buffer constructor, it had to replaced with Buffer::create